posthog/models/team.py

```diff
@@ -99,6 +99,9 @@ class Team(models.Model):
     is_demo: models.BooleanField = models.BooleanField(default=False)
     test_account_filters: JSONField = JSONField(default=list)
     timezone: models.CharField = models.CharField(max_length=240, choices=TIMEZONES, default="UTC")
+    important_data_attributes: ArrayField = ArrayField(
```

Comment:
Note: This same model has used `JSONField` before for this w/ the event_names etc columns. I think it's worth being consistent.

Reply:
Makes sense. I also renamed it from `important_data_attributes` to just `data_attributes`. `important` is not a very informative prefix.